pkg/apiserver/apiserver.go

```diff
@@ -187,7 +185,7 @@ func (s *APIServer) buildHandlerChain() {
 	handler = filters.WithKubeAPIServer(handler, s.KubernetesClient.Config(), &errorResponder{})
 	handler = filters.WithMultipleClusterDispatcher(handler, dispatch.DefaultClusterDispatch)
 
-	excludedPaths := []string{"/oauth/authorize", "/oauth/token"}
+	excludedPaths := []string{"/oauth/*", "/server/configs/*"}
```

Comment:
I suggest we use `/kapis/configz` to respond for components query, `/kapis/configz/oauth` for oauth configuration

Reply:
I don't think so, `configz` is more like the non resource API, kubernetes API prefixes or kubesphere API prefixes  is not needed. Even if configz is regarded as a resource, the API should conform to the specification: https://kubernetes.io/docs/reference/using-api/api-concepts/#standard-api-terminology.
I think `/kapi/v1alpha1/configs/configz`,  `/kapi/v1alpha1/configs/oauth` is better, it's 
easy to migrate to CRD.